Tasks/frmTaskManager.cs

```diff
@@ -13,91 +13,46 @@
 
```

Comment:
Hesitant with Task Manager, as I recently implomented a new feature and I need to treat everything carefully  so it doesn't break.